kilkari/src/main/java/org/motechproject/nms/kilkari/service/impl/MctsBeneficiaryImportServiceImpl.java

```diff
@@ -303,6 +303,12 @@ public ChildImportRejection importChildRecord(Map<String, Object> record, Subscr
 
                 try {
                     MctsMother motherInstance = (MctsMother) motherRecord;
+                    //changes made to remove special character form the mcts child's mother_id
+                    String mctsMotherId=motherInstance.getBeneficiaryId();
+                    mctsMotherId=mctsMotherId.replaceAll("[\\n\\t\\r ]","");
+                    motherInstance.setBeneficiaryId(mctsMotherId);
+                    record.replace(KilkariConstants.MOTHER_ID,motherInstance);
+
```

Comment:
This should also be using the common method.

Reply:
> This should also be using the common method.

**to replace this line**
 String mctsMotherId=motherInstance.getBeneficiaryId();
 mctsMotherId=mctsMotherId.replaceAll("[\\n\\t\\r ]","");

**we have to add these much of line** 

String mctsMotherId=motherInstance.getBeneficiaryId();
Map<String, Object> motherRecordTemp= new HashMap<>();;
motherRecordTemp.put("ID_No",mctsMotherId);
ArrayList<String > listOfIds=new ArrayList<>();
listOfIds.add("ID_No");
removeSpecialChar(listOfIds,record);
mctsMotherId=(String)motherRecordTemp.get("ID_No");